.gitignore

```diff
@@ -18,9 +18,10 @@
 *.swp
 *.swo
 *~
-
+bin/
```

Comment:
How are those changes related to updating Go modules?

Reply:
this is because the verify-boilerplate downloads the checker in this directory